pkgs/by-name/co/commitlint-rs/package.nix

```
@@ -0,0 +1,30 @@
+{
+  fetchFromGitHub,
+  lib,
+  nix-update-script,
+  rustPlatform,
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "commitlint-rs";
+  version = "0.1.11";
+
+  src = fetchFromGitHub {
+    owner = "KeisukeYamashita";
+    repo = pname;
```

```suggestion
    repo = "commitlint-rs;
```
https://github.com/NixOS/nixpkgs/issues/277994